ddtrace/tracer/option.go

```diff
@@ -138,6 +138,23 @@ type config struct {
 
 	// enabled reports whether tracing is enabled.
 	enabled bool
+
+	// postProcessor holds the function used to process finished spans of a trace.
+	// It reports whether the trace should be dropped.
+	postProcessor func([]ddtrace.ReadWriteSpan) bool
+}
+
+// WithPostProcessor enables processing finished spans of a trace by f. Spans
+// can be read from and written to in the processor by using the methods specified by
+// the ReadWriteSpan interface. f reports whether the trace should be dropped (dropped
+// traces and spans will not be accounted for in APM stats). Please note that modifying
```

Comment:
> dropped traces and spans will not be accounted for in APM stats

This will not be true anymore once client side stats are GA.

Reply:
Can you make sure the "Please note ..." part starts on a new paragraph, with an empty line between them?